api/client/build.go

```diff
@@ -209,6 +210,15 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 		}
 	}
 
+	var shmSize int64 = 67108864 // initial SHM size is 64MB
```

Comment:
yes, I was thinking something like that. (I think we used `-1` for other options, but I'm not too fond of "magic values")

@tiborvass @jfrazelle any preference here?